src/EventListener/StatsSubscriber.php

```diff
@@ -59,10 +59,12 @@ private function isRoutedFromHome(Request $request): bool
      */
     public function persistData(Request $request, $controller): bool
     {
-        if (null === ($repository = $request->get('repository', null)) || $this->isRoutedFromHome($request)) {
+        $referer = $request->headers->get('referer');
+        if (null === ($repository = $request->get('repository', null)) ||
+            $this->isRoutedFromHome($request) ||
+            $request->getSchemeAndHttpHost() === $referer) {
```

Comment:
it would not be better to use a `strpos`?

```php
strpos($referer, $request->getSchemeAndHttpHost()) !== false
```

Reply:
We can have the case that referer is not equal but contained?

Reply:
I think so! Example, the presence or absence of the final slash... you know?

Reply:
yes, make sense.

Reply:
I'm expecting `$request->getSchemeAndHttpHost` will return scheme and host, not the path ... no?  .. so no slash in it. Am I missing the point?

Reply:
it returns the full path.